catalog/resource_sql_table.go

```diff
@@ -89,6 +99,9 @@ func (ti SqlTableInfo) CustomizeSchema(s *common.CustomizableSchema) *common.Cus
 	s.SchemaPath("column", "type").SetCustomSuppressDiff(func(k, old, new string, d *schema.ResourceData) bool {
 		return getColumnType(old) == getColumnType(new)
 	})
+	s.SchemaPath("constraint", "type").SetCustomSuppressDiff(func(k, old, new string, d *schema.ResourceData) bool {
+		return getColumnType(old) == getColumnType(new)
```

Comment:
This would work because we are essentially checking for case insensitivity and ignoring the mapping here:
```
	if alias, ok := columnTypeAliases[caseInsensitiveColumnType]; ok {
		return alias
	}
```

We shouldn't use getColumnType here.